requirements/verification-by-reference.md

```diff
@@ -33,14 +33,14 @@ This doc experiments with the avoidance of signatures as the reference.
 1. Work within the constraints of the existing OCI-distribution spec api set.
    Supporting Notary v1, or Docker Content Trust, requires new APIs.
    APIs are comparatively cheap to implement atop the persistence stores of registries.
-   New APIs should be part of the Notary v2 spec, which may represent changes to the OCI-distribution spec or implementations of a new extension model over the Distribution spec.
+   New APIs should be part of the Notary Project spec, which may represent changes to the OCI-distribution spec or implementations of a new extension model over the Distribution spec.
 1. Compatibility with Notary v1.
    Registries that have implemented Notary v1 are looking for a better solution.
-   We cumulatively have little existing usage, and if successful, we expect all customers would rapidly move to Notary v2 eliminating the need to maintain two sets of APIs for an extended period of time.
+   We cumulatively have little existing usage, and if successful, we expect all customers would rapidly move to Notary Project eliminating the need to maintain two sets of APIs for an extended period of time.
```

Comment:
We cumulatively have little existing usage, and if successful, we expect all customers would rapidly move to **the** Notary Project